.github/workflows/on_PR.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 
       - name: Install requirements
         run: |
-          python3 -m pip install -r ./scripts/ci/requirements.txt && sudo apt install uncrustify
+          python3 -m pip install -r ./scripts/ci/requirements.txt && sudo apt-get install clang-format
```

Comment:
From my experience clang-format behaves differently on different version, we should consider settling for one specific version of clang-format